packages/toolkit/UPGRADING.md

```diff
@@ -2,6 +2,20 @@
 
 ## Upgrading from 5 to 6
 
+There are a few major changes in this release. While it probably won't break your build, it's worth noting everything that's changed.
+
+* Node version required of 18 (previous LTS)
+* Swapping `postcss-editor-styles` with `postcss-editor-styles-wrapper` which is compatible with PostCSS 8. This should be a seamless change as this is a fork of the original package that was updated to be compatible with PostCSS 8 and fixes wrong behaviour in some circumstances.
```

Comment:
This is a breaking change for those extending the postcss.config.js file so they will need to update it.

Reply:
Good catch